### PR TITLE
Support of Russian and English languages

### PR DESCRIPTION
**Pull request by takeAmoment**
1.  Using i18 next
2. When  the position of the switcher changes, the language of page changes to the selected language
3. Saving the selected language in local storage;

Screen Shot 2022-11-20 at 12 52 46

### CODE REVIEW

File: src/components/header/header.tsx [hunk 1]
```diff
@@ -109,18 +123,22 @@ const HeaderOfApp = () => {
...
           </Menu.SubMenu>
         )}
       </Menu>
       <div className={token ? styles.language__blockToken : styles.language__block}>
         <Space>
           <Text className={styles.language}>En</Text>
-          <Switch className={styles.language__switcher} defaultChecked onChange={onChange} />
+          <Switch
+            className={styles.language__switcher}
+            defaultChecked={lang === 'en' ? false : true}
```
Comment (uladzimir-a) [inline src/components/header/header.tsx]:
```suggestion
            defaultChecked={lang !== 'en'}
```
можно упростить

File: src/components/header/header.tsx [hunk 2]
```diff
@@ -20,8 +22,15 @@ const { Text } = Typography;
 const HeaderOfApp = () => {
   const location = useLocation();
   const token = useAppSelector((state) => state.auth.userData?.token);
+  const langFromStorage = localStorage.getItem('lang');
 
   const [open, setOpen] = useState<boolean>(false);
+  const [lang, setLang] = useState<string>(langFromStorage ? langFromStorage : 'en');
```
Comment (uladzimir-a) [inline src/components/header/header.tsx]:
```suggestion
  const [lang, setLang] = useState<string>(langFromStorage ?? 'en');
```
тоже можно упростить

File: src/translation/en.json [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+    "Header welcome link": "Welcome",
+    "Header board link": "Board",
+    "Header login link": "Login",
+    "Header sign up link": "Sign up",
+    "Header edit profile link": "Profile",
+    "Header logout link": "Logout",
+    "Welcome title": "Welcome to Teamber",
+    "Welcome text": "Teamber is a new unique application that will help you manage your workflow more effective. Create tasks, work with your team and reach your goals.\"Coming together together is the beginning. Keeping together is progress. Working together is success.\" - Henry Ford",
+    "Welcome button": "Let's start",
+    "Avantages title": "Our advantages",
```
Comment (uladzimir-a) [inline src/translation/en.json]:
```suggestion
    "Advantages title": "Our advantages",
```

File: src/translation/ru.json [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+    "Header welcome link": "Добро пожаловать",
+    "Header board link": "Доски",
+    "Header login link": "Войти",
+    "Header sign up link": "Регистрация",
+    "Header edit profile link": "Профиль",
+    "Header logout link": "Выйти",
+    "Welcome title": "Добро пожаловать в Teamber",
+    "Welcome text": "Teamber - новое уникальное приложение которое поможет управлять рабочим процессом более эффективно. Создавай задания, работай со своей командой и достигай целей.\"Собраться вместе - это начало. Держаться вместе - прогресс. Работать вместе - успех.\" - Генри Форд",
+    "Welcome button": "Начать",
+    "Avantages title": "Наши преимущества",
```
Comment (uladzimir-a) [inline src/translation/ru.json]:
```suggestion
    "Advantages title": "Наши преимущества",
```

**Review by olgad21 — APPROVED** [review]

**Review by uladzimir-a — APPROVED** [review]

**Pull request merged**